go/test/endtoend/vreplication/cluster.go

```diff
@@ -211,7 +207,7 @@ func (vc *VitessCluster) AddKeyspace(t *testing.T, cells []*Cell, ksName string,
 	keyspace.VSchema = vschema
 	for _, cell := range cells {
 		if len(cell.Vtgates) == 0 {
-			fmt.Println("Starting vtgate")
+			t.Logf("Starting vtgate")
```

Comment:
Do we still get logs to console with this change?
I usually run `go test -alsologtostderr -v`.